delete-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusDeleteHandlerTest.java

```diff
@@ -0,0 +1,194 @@
+
+    public static final Context CONTEXT = mock(Context.class);
+    public static final RequestParametersEntity EMPTY_REQUEST_PARAMETERS = null;
+    public static final ResponseElementsEntity EMPTY_RESPONSE_ELEMENTS = null;
+    public static final UserIdentityEntity EMPTY_USER_IDENTITY = null;
+    public static final long SOME_FILE_SIZE = 100L;
+
+    public static final String DELETE_MULTIPLE_IDENTIFIERS_FILE = "deleteMultipleIdentifiers.txt";
+    public static final String DELETE_SINGLE_IDENTIFIER_FILE = "deleteSingleIdentifier.txt";
+    public static final String HARDCODED_IDENTIFIER_IN_DELETE_SINGLE_FILE = "0000687314";
+    public static final List<String> HARDCODED_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE =
+            Arrays.asList("0000687314", "0017752991", "33947568076", "85084744636");
+
+    private FakeS3Client s3Client;
+    private S3Driver s3Driver;
+    private ScopusDeleteHandler scopusDeleteHandler;
+    private FakeEventBridgeClient fakeEventBridgeClient;
+
+
+    @BeforeEach
+    public void init() {
+        s3Client = new FakeS3Client();
+        s3Driver = new S3Driver(s3Client, "ignoredValue");
+        fakeEventBridgeClient = new FakeEventBridgeClient();
+        scopusDeleteHandler = new ScopusDeleteHandler(s3Client, fakeEventBridgeClient);
+    }
+
+    @Test
+    void shouldEmitEventWithSingleScopusIdentifierWhenInputFileContainsSingleScopusIdentifier() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_SINGLE_IDENTIFIER_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        assertThat(fakeEventBridgeClient.getRequestEntries().size(), is(equalTo(0)));
+        scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        var actualIdentifiersFromEventBodies = getIdentifiersFromEventBodies();
+        assertThat(actualIdentifiersFromEventBodies.size(), is(equalTo(1)));
+        assertThat(actualIdentifiersFromEventBodies.get(0),
+                is(equalTo(HARDCODED_IDENTIFIER_IN_DELETE_SINGLE_FILE)));
+    }
+
+    @Test
+    void shouldEmitEventsWithScopusIdentifiersWhenInputFileContainsMultipleScopusIdentifiers() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(DELETE_MULTIPLE_IDENTIFIERS_FILE));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        var s3Event = createS3Event(uri);
+        scopusDeleteHandler.handleRequest(s3Event, CONTEXT);
+        var actualIdentifiersFromEventBodies = getIdentifiersFromEventBodies();
+        assertThat(actualIdentifiersFromEventBodies.size(),
+                is(equalTo(HARDCODED_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.size())));
+        assertThat(actualIdentifiersFromEventBodies,
+                containsInAnyOrder(HARDCODED_IDENTIFIERS_IN_DELETE_MULTIPLE_FILE.toArray()));
+    }
+
+    @Test
+    void shouldLogExceptionMessageWhenExceptionOccursInS3Client() {
```

Comment:
Recently I have been getting a warning from SonarLint in Intellij that it is not a good idea to log  the Exception and re-throw because then you are logging twice. We should test this theory because we are paying in Cloudwatch by volume